pkg/controllers/dynakube/extension/eec/statefulset.go

```diff
@@ -312,7 +312,7 @@ func buildContainerVolumeMounts(dk *dynakube.DynaKube) []corev1.VolumeMount {
 		})
 	}
 
-	if dk.Spec.ActiveGate.TlsSecretName != "" {
+	if dk.ActiveGate().HasCaCert() {
```

Comment:
Question:
`dk.ActiveGate().HasCaCert()` is for which Cert, it is(currently) only for the `tlsCert`? 
- A saw you added `SetTrustedCAs` to the `activegate` but you don't use `ag.trustedCAs` (other then setting it)

Did you wanted to be something like:
```go
func (ag *Spec) HasCaCert() bool {
	return ag.IsEnabled() && (ag.TlsSecretName != "" || ag.IsAutomaticTlsSecretEnabled() || ag.ag.trustedCAs != "")
}
```

(because you are setting a volume called `activeGateTrustedCertVolumeName`, so I am guessing, that it should also exist if only `trustedCAs` were given)

Reply:
Extensions needs `tlsSecretName` only. So no changes are needed.
But `trustedCAs` will be removed from `activegate.Spec` because is not needed.